src/Toml/Type/AnyValue.hs

```diff
@@ -31,7 +33,12 @@ instance Show AnyValue where
 instance Eq AnyValue where
     (AnyValue val1) == (AnyValue val2) = case sameValue val1 val2 of
         Right Refl -> val1 == val2
-        Left _ -> False
+        Left _     -> False
+
+instance NFData AnyValue where
+    rnf = rwhnf
+
+instance Generic AnyValue
```

Comment:
Heh.. Deriving of `Generic` is not possible for existential types like `AnyValue`. So let's remove this instance.